CONTRIBUTING.rst

```diff
@@ -57,7 +57,7 @@ Follow these steps to submit your first pull request to ``cloud-init``:
 * Configure ``git`` with your email and name for commit messages.
 
   Your name will appear in commit messages and will also be used in
-  changelogs or release notes. Give yourself credit! ::
+  change logs or release notes. Give yourself credit! ::
```

Comment:
changelog is usually written as a single word (rather than change log) - I would say the original is correct in this case